File: totosuki/ABC/B/037.py
N, Q = map(int, input().split())
rslt = [0] * N

for _ in range(Q):
  L, R, T = map(int, input().split())
  for i in range(L-1, R):
    rslt[i] = T

[print(r) for r in rslt]

File: OCH_Tao/ABC/A/037.js
const input = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(str => parseInt(str, 10));
if (input[0] < input[1]) {
  console.log(Math.trunc(input[2] / input[0]));
} else {
  console.log(Math.trunc(input[2] / input[1]));
}